streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/SavePointServiceImpl.java

```diff
@@ -114,6 +128,41 @@ public SavePoint getLatest(Long id) {
     return this.getOne(queryWrapper);
   }
 
+  @Override
+  public void trigger(Long appId, @Nullable String savepointPath) {
+    log.info("Start to trigger savepoint for app {}", appId);
+    Application app = appService.getById(appId);
+    AssertUtils.notNull(app, String.format("The application %s doesn't exist.", appId));
+    String trackingUrl = app.getJobManagerUrl();
+    AssertUtils.state(
+        StringUtils.isNotEmpty(trackingUrl),
+        String.format("The flink trackingUrl for app[%s] isn't available.", appId));
+    AssertUtils.state(
+        StringUtils.isNotEmpty(app.getJobId()),
+        String.format("The jobId of application[%s] is absent.", appId));
+    String triggerId = triggerSavepoint(trackingUrl, app, savepointPath);
+    log.info("Request savepoint successful in triggerId {}", triggerId);
+  }
+
+  private String triggerSavepoint(
```

Comment:
I suggest move this method in `FlinkSubmitter`, and the service layer of the platform just call it.

Reply:
updated. Thx for the review.